content/services/storage/storage-tool.yaml

```
@@ -228,7 +228,7 @@ services:
       - name: storage_warmth
         class: cold
 
-  - name: LabArchives
+  - name: Lab_Archives
```

Are we removing work separators in this file? Looks like the `Campus File Storage Non-replicated` is now spaces

```suggestion
  - name: Lab Archives
```

---

i copied and pasted the OG storage yaml to make sure it is consistent with what the storage team has worked with before. Removes some of these changes that I had made while debugging